src/test/test-execute.c

```diff
@@ -381,18 +383,33 @@ static void test_exec_systemcallfilter(Manager *m) {
         test(m, "exec-systemcallfilter-not-failing2.service", 0, CLD_EXITED);
         test(m, "exec-systemcallfilter-failing.service", SIGSYS, CLD_KILLED);
         test(m, "exec-systemcallfilter-failing2.service", SIGSYS, CLD_KILLED);
+
+        r = find_binary("python3", NULL);
```

Comment:
`python` is used in some tests to just call `uname` and return the errno. Maybe, instead of skipping those tests, we could write a small binary doing exactly that. This way we could get rid of `python` entirely. More importantly, it would be easier to control what this binary is doing.